File: URI_1036_bhaskaras_formula.js
const lines = ["0.0 20.0 5.0"];

const inputArr = lines[0].split(" ");
let a = Number(inputArr[0]);
let b = Number(inputArr[1]);
let c = Number(inputArr[2]);

if (a === 0.0 || 4 * a * c > b * b) {
    console.log("Impossivel calcular");
} else {
    const r1 = (-b + Math.sqrt(b * b - 4 * a * c)) / (2 * a);
    const r2 = (-b - Math.sqrt(b * b - 4 * a * c)) / (2 * a);

    console.log("R1 = " + r1.toFixed(5));
    console.log("R2 = " + r2.toFixed(5));
}

File: URI_1036_bhaskaras_formula.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class URI_1036_bhaskaras_formula {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		double a = scanner.nextDouble();
		double b = scanner.nextDouble();
		double c = scanner.nextDouble();

		if (a == 0.0 || 4 * a * c > b * b) {
			System.out.println("Impossivel calcular");
		} else {
			double r1 = (-b + Math.sqrt(b * b - 4 * a * c)) / (2 * a);
			double r2 = (-b - Math.sqrt(b * b - 4 * a * c)) / (2 * a);

			DecimalFormat df = new DecimalFormat("0.00000");
			System.out.println("R1 = " + df.format(r1));
			System.out.println("R2 = " + df.format(r2));
		}

		scanner.close();
	}
}
